VimeoNetworking/Sources/NSError+Extensions.swift

```diff
@@ -192,20 +191,20 @@ public extension NSError
         /// Returns the api error code of the failing response, if available
     public var vimeoServerErrorCode: Int?
     {
-        if let errorCode = (self.userInfo[self.dynamicType.VimeoErrorCodeKeyLegacy] as? NSNumber)?.integerValue
+        if let errorCode = (self.userInfo[type(of: self).VimeoErrorCodeKeyLegacy] as? NSNumber)?.intValue
         {
             return errorCode
         }
         
         if let response = self.userInfo[AFNetworkingOperationFailingURLResponseErrorKey],
-            let headers = response.allHeaderFields,
-            let vimeoErrorCode = headers[self.dynamicType.VimeoErrorCodeHeaderKey] as? String
+            let headers = (response as AnyObject).allHeaderFields,
```

Comment:
It took me a while to figure out what the hell was going on here. And I have a guess as to why it works, but that's still kind of nuts that it _does_.